### DISCUSSION TITLE
editor 里面支持修改字体吗

### DISCUSSION THREAD

**Discussion by coulis1018** [opensumi/core#1864] [Q&A] [answered]
如何去修改编辑器里面字体，大小，行间距等属性呢？

**Comment by erha19** [comment, marked as answer]
@coulis1018 可以在设置面板设置编辑器的一些基础设置，左下角的按钮可以点击进入设置面板，也可以用快捷键 `CMD + ,`, Windows 下是 `Ctrl + ,`
<img width="239" alt="image" src="https://user-images.githubusercontent.com/9823838/198280660-70c9dd3f-0dd1-40b4-a754-f51dc5e88b1d.png">
